index.html

```diff
@@ -59,12 +59,12 @@ <h2 id ="heading">My Recent Works</h2>
         </div>
         <div class="work-grid">
```

Comment:
- [ ] Great job adding your project modals dynamically, however you also need to add your project cards themselves dynamically as you did for the project modal.
- [ ] I kindly suggest that you remove the project cards you added manually in your HTML and add them dynamically through JavaScript as you did for the modal

<img width="782" alt="image" src="https://user-images.githubusercontent.com/52934128/224064775-23c12fec-bbd0-4777-a8c5-54d0a7c15178.png">

Reply:
Do you want me to create object in javascript rather than getting information from main html file

Reply:
> what do i need to change?

@vikas924, you need to remove your projects from your HTML page, the create it dynamically through Javascript. Your HTML code will look like this after:

 A way to to this is to create a function that will display all the contents in the works section. eg
 
 ```
 function populateProjets() {
  the targetted works innerHTML = `all of it content........`
 }
 populateProjects();
 ```
 
 * The section should look something similar to this in the HTML file
 
 ```
 <section class="work-container">
         <!-- displayed dynamically with JS  -->
</section>
 ```

Reply:
sir, i have section which getting created dynamically and its path is
<section id="content">
      < inside showing throught javascript its not in html>
</section>
so i m not getting what do i need to change

Reply:
<section id="content">
        <!-- displayed dynamically with JS  -->
</section>
its under header not section with class section

Reply:
kindly, check section with id = content its changing with javascipt

Reply:
> Do i need to remove projects that were already in html before implementing popup and show those with javascript also

@vikas924 your projects are not implemented dynamically and yes you have to remove the projects already in html and add those projects through javascript dynamically